### ISSUE TITLE
Algorithmic performance and complexity: O(0)

### ISSUE THREAD

**Issue by rvzzz** [kelseyhightower/nocode#1733]
Proof:
![](https://www.mathsisfun.com/calculus/images/lim-1-x.png)
Where **_x_** approaching infinity represents the endless possibilities.
 ∴
![](https://saylordotorg.github.io/text_elementary-algebra/section_06/c80b7268ac8cc2cc6bcd5fbdf99e6072.jpg)

**Comment by phenax** [comment]
I think with a few more optimizations, we can do better than O(0)

**Comment by KernelDeimos** [comment]
Actually, see my reply in issue #1668 - I believe the time complexity of nocode is actually O(` `).

**Comment by rvzzz** [comment]
good point @phenax and @kernelDeimos ...I think this needs more research indeed... And I hope someday Donald Knuth will look at this repo and it inspires him to work on **_the art of computer programming - volume 5_**. Although I have the feeling he knew about this theory before but didn't want to publish it. (I think the NSA has something to do with it.)
